src/PlatformChecker.ts

```diff
@@ -0,0 +1,3 @@
+export function isJest(): boolean {
+  return Boolean(process.env.JEST_WORKER_ID);
```

Comment:
Wouldn't a `process.env.JEST_WORKER_ID != null` be more semantically correct? Does the linter say anything about this?

Reply:
No linter complaints, but agreed that `!= null` captures the meaning better.